pwd-service/src/main/java/org/pwd/domain/contact/HtmlEmailMessage.java

```diff
@@ -0,0 +1,37 @@
+package org.pwd.domain.contact;
+
+import com.lyncode.jtwig.JtwigModelMap;
+import com.lyncode.jtwig.JtwigTemplate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author Sławomir Mikulski
+ */
+public class HtmlEmailMessage extends EmailMessage {
+    private static final Logger logger = LoggerFactory.getLogger(EmailMessage.class);
+    private final String html;
+
+    public HtmlEmailMessage(String from, String to, String subject, JtwigTemplate jtwigTemplate, JtwigModelMap modelMap) {
+        this.from = from;
+        this.to = to;
+        this.subject = subject;
+        this.html = renderTemplateIntoText(jtwigTemplate, modelMap);
+    }
+
+    public String getHtml() {
+        return html;
+    }
+
+    private String renderTemplateIntoText(JtwigTemplate jtwigTemplate, JtwigModelMap modelMap) {
+        String result;
+        try {
+            result = jtwigTemplate.output(modelMap);
+        }
+        catch (Exception ex){
+            result = "";
```

Comment:
wydaje mi sie ze to zachowanie jest niepoprawne. Wyslesz maila bez tresci kiedy template sie nie wyrenderuje, raczej nie polykajmy tego wyjatku tylko rzucmy jakis runtime